src/app/pie-chart/pie-chart.component.ts

```diff
@@ -0,0 +1,45 @@
+import {Component, OnInit} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+
+@Component({
+  selector: 'app-pie-chart',
+  templateUrl: './pie-chart.component.html',
+  styleUrls: ['./pie-chart.component.css']
+})
+export class PieChartComponent {
+  constructor(private http: HttpClient) {
+  }
+
+  public chartType = 'pie';
+  url = 'http://10.3.4.114:4200/Gradle___Alor_Backend_war/rest/mdm/dist';
```

Comment:
same goes here: everytime you write the same code for the second time, find an abstraction for it.